x-pack/plugins/observability/public/components/shared/exploratory_view/embeddable/embeddable.tsx

```diff
@@ -19,19 +19,31 @@ import { ReportConfigMap } from '../contexts/exploratory_view_config';
 import { obsvReportConfigMap } from '../obsv_exploratory_view';
 import { ActionTypes, useActions } from './use_actions';
 import { AddToCaseAction } from '../header/add_to_case_action';
+import { ViewMode } from '../../../../../../../../src/plugins/embeddable/common';
+import { observabilityFeatureId } from '../../../../../common';
+import { SingleMetric } from './single_metric';
 
 export interface ExploratoryEmbeddableProps {
-  reportType: ReportViewType;
-  attributes: AllSeries;
+  appId?: 'security' | 'observability';
   appendTitle?: JSX.Element;
+  attributes?: AllSeries;
   title: string | JSX.Element;
   showCalculationMethod?: boolean;
   axisTitlesVisibility?: XYState['axisTitlesVisibilitySettings'];
-  legendIsVisible?: boolean;
+  customHeight?: string | number;
+  customLensAttrs?: any;
+  customTimeRange?: { from: string; to: string };
   dataTypesIndexPatterns?: Partial<Record<AppDataType, string>>;
+  isSingleMetric?: boolean;
+  legendIsVisible?: boolean;
+  metricIcon?: string;
+  metricIconColor?: string;
+  metricPostfix?: string;
+  onBrushEnd?: (param: { range: number[] }) => void;
+  owner?: string;
```

Comment:
i think owner and appId are essentially same? no. we can probably remove one of them to improve things.

Reply:
For Security Solution it's a bit different between `appId` and `owner`, the appId is `securitySolutionUI`, but owner is `securitySolution`. It would be useful for Security Solution if we could keep both props.

Reply:
Sure, makes sense